cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,25 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
```

Comment:
The `before` hook clears the database before running the tests. Ensure that the `db:clear` task is correctly implemented and does not interfere with any necessary setup for the tests. Consider verifying the database state after clearing to ensure consistency across tests.